examples/resources/spectrocloud_cluster_vsphere/resource.tf

```diff
@@ -1,18 +1,19 @@
 data "spectrocloud_cluster_profile" "vmware_profile" {
   name    = "vmware-public-repo"
-  version = "2.0.0"
+  version = "1.0.0"
   context = "tenant"
 }
 data "spectrocloud_cloudaccount_vsphere" "vmware_account" {
-  name = "gm-pcg-wop-d"
+  name = var.shared_vmware_cloud_account_name
```

Comment:
@SivaanandM could you please provide complementary change in variables.tf?

Reply:
usually, we won't have a custom variable defined in variable.tf, we used to maintain in tfvars. I will revert back to hard values